### DISCUSSION TITLE
Any way to use unpoly with worpdress?

### DISCUSSION THREAD

**Discussion by stratboy** [unpoly/unpoly#450] [Q&A]
I  think Unpoly, HTMX and Turbo approaches are the future. Any way to use Unpoly with wordpress? Did any of you tried a bit?

**Comment by triskweline** [comment]
Unpoly works with any backend that can produce HTML. That includes Wordpress!

No server-side intergration is required, just [include Unpoly](https://unpoly.com/install) in your JavaScripts and start using attributes like [`[up-follow]`](https://unpoly.com/a-up-follow).

**Reply by stratboy** [reply]
:)
